src/main/java/net/minecraftforge/fluids/FluidRegistry.java

```diff
@@ -356,6 +356,21 @@ public static String getDefaultFluidName(Fluid key)
         return name;
     }
 
+    @Nullable
+    public static String getModId(@Nullable FluidStack fluidStack)
+    {
+        if (fluidStack != null)
```

Comment:
`Optional` is fine for return values but using it for parameters is discouraged.
It would make the implementation look neater but it doesn't improve anything for people calling the method.